lib/daru/io/io.rb

```diff
@@ -221,6 +227,25 @@ def from_csv_hash(path, opts)
 
         headers.each_with_index.map { |h, i| [h, csv_as_arrays[i]] }.to_h
       end
+
+      def from_html_get_list(page)
+        page.search('table').map { |table|
+          index = table.search('th').map { |header| header.text.strip }
+          data = table.search('tr').map { |row| row.search('td').map { |val| val.text.strip } }.keep_if { |a| a!=[] }
```

Comment:
Yeah, definitely - support for more types of table types and tag structures is required. I'll also have a look at `pandas` `from_html` module and discuss again regarding this.